gen/MotController_data.yml

```diff
@@ -794,6 +827,41 @@ c_MotController_ConfigMaxIntegralAccumulator:
         success and report an error if it times out. If zero, no
         blocking or checking is performed.
     :returns: Error Code generated by function. 0 indicates no error.
+c_MotController_ConfigClosedLoopPeakOutput:
+  code: |
+    self.hal_data['profile%d_peakOutput' % slotIdx] = percentOut
+  doc: |
+    Sets the peak closed-loop output.  This peak output is slot-specific and
+    is applied to the output of the associated PID loop.
+    This setting is seperate from the generic Peak Output setting.
+    
+    :param slotIdx:    Parameter slot for the constant.
+    
+    :param percentOut: Peak Percent Output from 0 to 1.  This value is absolute and
+                       the magnitude will apply in both forward and reverse directions.
+    
+    :param timeoutMs:  Timeout value in ms. If nonzero, function will wait for
+                       config success and report an error if it times out.
+                       If zero, no blocking or checking is performed.
+    
+    :returns: Error Code generated by function. 0 indicates no error.
+c_MotController_ConfigClosedLoopPeriod:
+  code: |
+    self.hal_data['profile%d_period' % slotIdx] = loopTimeMs / 1000.0
```

Comment:
I guess this should be used by the simulation PID loop, probably.

Reply:
aw, man did they make that configurable?